Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM composer
+
+FROM webdevops/php-nginx:7.2
+
+ENV WEB_DOCUMENT_ROOT=/app/web
+ENV COMPOSER_ALLOW_SUPERUSER 1
+
+ENV PACKAGES_NAME='Terramar Labs'
+ENV PACKAGES_HOMEPAGE='https://github.com/terramar-labs/packages'
+ENV PACKAGES_CONTACT='contact@terramarlabs.com'
+ENV PACKAGES_BASEPATH='https://localhost'
+ENV PACKAGES_SECURE=false
+ENV PACKAGES_USER=user
+ENV PACKAGES_PASSWORD=password
+ENV PACKAGES_PDO_DRIVER=pdo_sqlite
+ENV PACKAGES_PDO_PATH=%app.root_dir%/database.sqlite
+ENV PACKAGES_PDO_DBNAME=packages
+ENV PACKAGES_REDIS_HOST='redis://redis'
+ENV PACKAGES_REDIS_PORT=6379
+
+WORKDIR /app
+
+COPY --from=0 /usr/bin/composer /usr/bin/composer
+
+RUN apt-get update \
```

Comment:
The webdevops image contains a helpful wrapper `apt-install gettext-base procps` which updates, installs and cleans afterwards.